src/utils/validations/login.js

```diff
@@ -31,7 +31,7 @@ export const lastName = (value) => {
 
 export const confirmPassword = (password, data) => {
   let helperText
-  if (password !== data.confirmPassword) {
+  if (password !== data.password) {
     helperText = 'common.errorMessages.passwordsDontMatch'
```

Comment:
helperText = emptyField(password, password !== data.password ? 'common.errorMessages.passwordsDontMatch' : '')

Reply:
refactored using ternar
now helpertext is a string anyway, either full or empty, 
so i had to change it to be string in all validations.